src/components/common/AuthWrapper.scss

```diff
@@ -0,0 +1,36 @@
+@import '../../styles/styles.scss';
+
+.formContainer {
+
+    display: grid;
+    justify-content: center;
+    margin: auto;
+    text-align: center;
+
+}
+
+.appImgContainer{
+    
+    &__mockAppImg {
+        
+        max-width:550px;
```

Comment:
Please use rem instead of pixel. It's one of our standards for these kind of measures.